requirements.txt

```diff
@@ -0,0 +1,4 @@
+Django
```

Comment:
Actually this should be specified with maximum version. Because if in the future version django will change their API we will have problem with installing it.